certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6000 @@
+            "type": "string",
+            "description": "Cloud Commander partner identifier.",
+            "x-ms-summary": "Cloud Commander partner identifier.",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "customerId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Cloud Commander customer identifier.",
+            "x-ms-summary": "Cloud Commander customer identifier.",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "tenantId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Microsoft tenant identifier of the customer.",
+            "x-ms-summary": "Microsoft tenant Id of the customer.",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "operationId": "GetSubscriptions"
+      }
+    },
+    "/partnercenter/v1/partners/{partnerId}/organizations/{customerId}/v1.0/customers/{tenantId}/subscriptions/{subscriptionId}": {
+      "patch": {
+        "responses": {
+          "200": {
+            "description": "Ok."
+          },
+          "401": {
+            "description": "The request was not authenticated. The client needs to authenticate with Cloud Commander APIs first."
+          },
+          "403": {
+            "description": "The request was not authorized. The client needs to have authorization to access this resource."
+          },
+          "500": {
+            "description": "Unexpected error."
+          }
+        },
+        "summary": "Update Partner Center subscription quantity",
+        "parameters": [
+          {
+            "name": "partnerId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Cloud Commander partner identifier.",
+            "x-ms-summary": "Cloud Commander partner id.",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "customerId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Cloud Commander customer identifier.",
+            "x-ms-summary": "Cloud Commander customer id.",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "tenantId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Microsoft tenant identifier of the customer.",
+            "x-ms-summary": "Microsoft tenant id of the customer.",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Subscription identifier of the customer.",
+            "x-ms-summary": "Subsription id.",
+            "x-ms-url-encoding": "single"
```

Comment:
Can you please correct the spelling "Subsription"

Reply:
Fixed